### PR TITLE
Add mixtral instruct model

### PR DESCRIPTION
**Pull request by novaturient95**
## mistralai-Mixtral-8x7B-Instruct-v01

**run to save model to blobstorage:** [link](https://ml.azure.com/runs/zen_soccer_dbp6tjndl9?wsid=/subscriptions/80c77c76-74ba-4c8c-8229-4c3b2957990c/resourcegroups/huggingface-registry-test1/workspaces/test-southcentralus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**benchmark run**: [link](https://ml.azure.com/experiments/id/025521ff-cd39-4cc2-9c72-d710131b882c/runs/sleepy_pocket_rwmqwv7ty8?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/arunrajora-dev-resource-group/providers/Microsoft.MachineLearningServices/workspaces/arun-distributed-tcn&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

**Online endpoint validation results**

ep: [link](https://ml.azure.com/endpoints/realtime/mistralai-mixtral-8x7b-instru-2/detail?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/model-validation-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)

image

**MLmodel**
```
flavors:
  hftransformersv2:
    code: null
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForCausalLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: mistralai/Mixtral-8x7B-Instruct-v0.1
    model_data: data
    pytorch_version: 1.13.1
    task_type: chat-completion
    transformers_version: 4.36.2
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.18
metadata:
  azureml.base_image: mcr.microsoft.com/azureml/curated/foundation-model-inference:27
  base_model_name: mistralai/Mixtral-8x7B-Instruct-v0.1
  base_model_task: text-generation
  is_acft_model: true
  is_finetuned_model: false
mlflow_version: 2.8.1
model_uuid: 1bb4326dd9b54966ba110733ceba112d
utc_time_created: '2024-01-03 14:28:27.890309'
```

**conda.yaml**
```
channels:
- conda-forge
dependencies:
- python=3.8.18
- pip<=23.3.1
- pip:
  - mlflow==2.8.1
  - cloudpickle==2.2.1
  - jsonpickle==3.0.2
  - mlflow-skinny==2.8.1
  - azureml-core==1.54.0.post1
  - azureml-mlflow==1.54.0.post1
  - azureml-metrics[all]==0.0.38.post1
  - scikit-learn==1.1.3
  - cryptography==41.0.7
  - python-dateutil==2.8.2
  - datasets==2.15.0
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.24.0
  - sentencepiece==0.1.99
  - transformers==4.36.2
  - torch==1.13.1
  - accelerate==0.25.0
  - Pillow==10.0.1
  - azureml-evaluate-mlflow==0.0.38
- pycocotools=2.0.4
name: mlflow-env

```

### CODE REVIEW

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+It does not have any moderation mechanisms. We're looking forward to engaging with the community on ways to
+make the model finely respect guardrails, allowing for deployment in environments requiring moderated outputs.
+
+# Inference samples
+
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
+
+# Sample inputs and outputs
```
Comment (novaturient95) [inline assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md:23]: use chat completion sample

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+For full details of this model please read [release blog post](https://mistral.ai/news/mixtral-of-experts/).
+
+# Limitations and Biases
+
+The Mixtral-8x7B Instruct model is a quick demonstration that the base model can be easily fine-tuned to achieve compelling performance.
+
+It does not have any moderation mechanisms. We're looking forward to engaging with the community on ways to
+make the model finely respect guardrails, allowing for deployment in environments requiring moderated outputs.
+
+# Inference samples
```
Comment (novaturient95) [inline assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md:17]: can remove this

Comment (novaturient95) [reply assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md]: Done

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md (hunk 1 above)
Comment (novaturient95) [reply assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/description.md:14]: Done

File: assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-Mixtral-8x7B-Instruct-v01
+path: ./
+properties:
+  SharedComputeCapacityEnabled: true
+  inference-min-sku-spec: 40|2|440|128
+  inference-recommended-sku: Standard_ND40rs_v2, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
```
Comment (novaturient95) [inline assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml:7]: validate on Standard_NC48ads_A100_v4

Comment (novaturient95) [reply assets/models/system/mistralai-Mixtral-8x7B-Instruct-v01/spec.yaml]: removed: Standard_NC48ads_A100_v4

**Review by svaruag — APPROVED** [review]

**Review by skasturi — APPROVED** [review]

**Pull request merged**